workshop-instructions/instructions.md

```diff
@@ -14,28 +15,37 @@
     - [Installing MPI for Parallel Simulations](#installing-mpi-for-parallel-simulations)
     - [Launch the GUI](#launch-the-gui)
 
+# Workshop Sign-In (Optional but Encouraged)
 
-# Cloning the HNN-Data Repository
-To follow along with the workshop, you'll need the [hnn-data repository](https://github.com/jonescompneurolab/hnn-data) on your local machine / virtual environment.
+Please consider 'signing in' to the workshop by providing your institution/location, position, and field of study. This information is used in reporting on milestones for the grants that continue to fund our dissemination and development. We don't ask for name or email, and the form should take no more than 30 seconds to complete.
 
-You can download the file [here](https://github.com/jonescompneurolab/hnn-data/archive/refs/heads/main.zip) or you can run the following command to clone the repository.
+Click [here](https://forms.gle/YnZX9ZRm8WvWpAGU7) to navigate to our Virtual Sign-In Form.
 
-```bash
-git clone https://github.com/jonescompneurolab/hnn-data.git
-```
+<p>Thanks for your support. &#128513;</p>
+
+
+# Download Workshop Materials
+To follow along with the workshop, you'll need to download the following materials:
 
-If you already cloned the hnn-data repository on your machine, be sure to update it to include the latest changes.
+- The [hnn-data repository](https://github.com/jonescompneurolab/hnn-data) (needed for the HNN-Core GUI tutorial)
+- The [hnn-workshop-materials repository](https://github.com/jasmainak/hnn-workshop-materials/tree/biomag_2024) (needed for the HNN-Core Python tutorial)
+
+Click [here](https://github.com/jonescompneurolab/hnn-data/archive/refs/heads/main.zip) to download the hnn-data folder, and [here](https://github.com/jasmainak/hnn-workshop-materials/archive/refs/heads/biomag_2024.zip) to download the hnn-workshop-materials folder. 
+
+Altenatively, you can clone the repositores with the follwing commands.
 
 ```bash
-cd hnn-data
-git pull
+git clone https://github.com/jonescompneurolab/hnn-data.git
+git clone https://github.com/jasmainak/hnn-workshop-materials.git
```

Comment:
I am working mostly on the colab as of now ... in two minds whether to abondon this altogether. But colab is really slow ...

Reply:
let's just drop this github url for now ... focus on colab. I'll add the github URL in the colab if it turns out to be something users can download